src/Symfony/Component/Messenger/Bridge/Kafka/composer.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "symfony/kafka-messenger",
+    "type": "symfony-bridge",
+    "description": "Symfony Messenger Kafka Transport Bridge",
+    "keywords": ["kafka", "symfony", "messenger", "transport", "queue", "bridge"],
+    "license": "MIT",
+    "require": {
+        "php": ">=7.2.5",
+        "ext-rdkafka": "^4.0",
```

Version 5.0 was released a few days ago, there were no important API changes, only PHP/librdkafka version support were dropped.
```suggestion
        "ext-rdkafka": "^4.0 | ^5.0",
```